readthedocs/core/signals.py

```diff
@@ -5,14 +5,19 @@
 import logging
 
 from corsheaders import signals
+from django.contrib.auth import get_user_model
+from django.db.models.signals import pre_delete
 from django.dispatch import Signal
-from django.db.models import Q
+from django.db.models import Q, Count
+from django.dispatch import receiver
 from future.backports.urllib.parse import urlparse
 
+from readthedocs.oauth.tasks import bulk_delete_oauth_remote_organizations
 from readthedocs.projects.models import Project, Domain
-
+from readthedocs.projects.tasks import bulk_delete_projects
 
 log = logging.getLogger(__name__)
+User = get_user_model()
```

Comment:
What's the reason for using get_user_model() here? The pattern we've used throughout the codebase is to import `from django.contrib.auth.models import User`

Reply:
I have gone through the documentation and it seems that using `settings.AUTH_USER_MODEL` is conventional way